app/src/main/scala/org/alephium/explorer/cache/BlockCache.scala

```diff
@@ -52,9 +52,11 @@ object BlockCache {
   }
 
   // scalastyle:off
-  def apply()(implicit groupSetting: GroupSetting,
-              ec: ExecutionContext,
-              dc: DatabaseConfig[PostgresProfile]): BlockCache = {
+  @SuppressWarnings(Array("org.wartremover.warts.DefaultArguments"))
+  def apply(cacheRowCountReloadTime: FiniteDuration = 10.seconds)(
```

Comment:
what about removing the default argument and add our reload times in our configuration?
If you prefer you can create a new task for this.